src/GitHub.VisualStudio/UI/Views/Controls/PullRequestListItem.xaml

```diff
@@ -110,6 +110,7 @@
                 Command="{Binding OpenPROnGitHub, RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type local:PullRequestListView}}}"
                 CommandParameter="{Binding Number}"
                 Content="{Binding Number}"
+                ToolTip="View Pull Request on GitHub"
```

Comment:
What about GitHub Enterprise instances? Usually we'd do something like:

1. For GitHub.com: `View Pull Request on GitHub"
2. For Enterprise: `View Pull Request on xyz.com` where `xyz.com` is the domain of the enterprise instance.